lib/db/mysql/index.js

```diff
@@ -374,8 +373,9 @@ MysqlStore.prototype = {
       return code;
     });
   },
-  removeCode: function removeCode(id) {
-    return this._write(QUERY_CODE_DELETE, [id]);
+  removeCode: function removeCode(code) {
+    var hash = encrypt.hash(code);
+    return this._write(QUERY_CODE_DELETE, [hash]);
```

Comment:
AFAICT these would not have worked in their original form, and must hash the code before trying to delete it.